tests/unit/transformers/test_datetime.py

```diff
@@ -442,6 +464,31 @@ def test__reverse_transform_datetime_format(self):
         expected = pd.Series(['Jan 01, 2020', 'Feb 01, 2020', 'Mar 01, 2020'])
         pd.testing.assert_series_equal(output, expected)
 
+    def test__reverse_transform_datetime_format_with_strftime_formats(self):
+        """Test the ``_reverse_transform`` method returns the correct datetime format.
+
+        Setup:
+            - Set the instance to have a different ``datetime_format`` which includes `-` on it.
+
+        Input:
+            - a numpy array of integers.
+
+        Output:
+            - a pandas ``Series`` of the datetimes in the right format.
+        """
+        # Setup
+        ute = UnixTimestampEncoder(missing_value_replacement=None)
+        ute.datetime_format = '%b %-d, %Y'
+        transformed = np.array([1.5778368e+18, 1.5805152e+18, 1.5830208e+18])
+
+        # Run
+        output = ute._reverse_transform(transformed)
+
+        # Assert
+        expected = pd.Series(['Jan 1, 2020', 'Feb 1, 2020', 'Mar 1, 2020'])
+        if 'windows' not in platform.system().lower():
```

Comment:
The code itself doesn't crash on windows right?

Reply:
Doesn't crash, however, it doesn't produce the expected `strftime`